helper.js

```diff
@@ -23,13 +23,12 @@ function lookupCoupon(itemno, callback) {
 
 function findSingleItemNo() {
```

Comment:
and for the singleItemNo function, it should be a lot easier to grab the ID from the URL... is this function used on a single item view page? where the url is ...%ITEM_IDZ%.html... ?